### PR TITLE
Translate "Update Ruby Meepup Information" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: #2818

Translate #3276, #3295
I don't touch translated terms like "미트업", "유저", we might improve these with seperate PR ;)

### CODE REVIEW

File: ko/community/user-groups/index.md [hunk 1]
```diff
@@ -19,24 +19,25 @@ Ruby 유저 그룹은 전적으로 Ruby에만 전념합니다. 일반적으로 
...
 
 ### 직접 그룹을 만들기
 
 직접 그룹을 만들기 원하시면 지역에 Ruby 그룹이 있는지를 먼저 확인하셔야 합니다.
-큰 그룹이 보통 더 유익하고 재미있으므로, 그 지역에 다른 Ruby 그룹이있다면 새로
-만드는 것은 최고의 선택은 아닐 것입니다.
-
-
+자신의 그룹을 시작하는 방법에 대한 조언을 구하고 싶다면,
+주최자 그룹을 방문해보세요.
```
Comment (yous) [inline ko/community/user-groups/index.md:37]:
```suggestion
미트업 주최자 그룹을 방문해보세요.
```

File: ko/community/user-groups/index.md [hunk 2]
```diff
@@ -19,24 +19,25 @@ Ruby 유저 그룹은 전적으로 Ruby에만 전념합니다. 일반적으로 
...
-
-[OnRuby][2]
-: OnRuby에서도 유저 그룹을 발견할 수 있습니다. OnRuby는 Ruby로 작성한 오픈 소스
-  플랫폼으로, 미트업을 만드는 데 사용할 수 있습니다. 소스는 [GitHub][3]에서 볼
-  수 있습니다.
+- [Ruby Meetup Groups on meetup.com][meetup]. Ruby 유저 그룹의 상당수는
+  Meetup 사이트를 홈페이지로 합니다. Meetup은 유저 그룹을 위한 몇 가지 툴을
+  제공하고 있습니다. 비공개 포럼, 공지를 위한 장소, 자동 리마인더, 훌륭한
+  RSVP(참석여부 체크) 시스템 등등이 그것이죠.
+- [rubyconferences.org/meetups][rc-meetups]는 전세계의 Ruby 미트업 이벤트 일람을
```
[the remark this answers is not in the document]
Comment (riseshia) [reply ko/community/user-groups/index.md:26]: Most of case, yes, they use Meetup, but there is some meetup not using Meetup, also. e.g. Bluegrass Ruby, Madison+ Ruby + NYC

File: ko/community/user-groups/index.md [hunk 3]
```diff
@@ -19,24 +19,25 @@ Ruby 유저 그룹은 전적으로 Ruby에만 전념합니다. 일반적으로 
...
-: Ruby 유저 그룹의 상당수는 Meetup 사이트를 홈페이지로 합니다. Meetup은
-유저 그룹을 위한 몇 가지 툴을 제공하고 있습니다. 비공개 포럼, 공지를 위한 장소,
-자동 리마인더, 훌륭한 RSVP(참석여부 체크) 시스템 등등이 그것이죠.
-
-[OnRuby][2]
-: OnRuby에서도 유저 그룹을 발견할 수 있습니다. OnRuby는 Ruby로 작성한 오픈 소스
-  플랫폼으로, 미트업을 만드는 데 사용할 수 있습니다. 소스는 [GitHub][3]에서 볼
-  수 있습니다.
+- [Ruby Meetup Groups on meetup.com][meetup]. Ruby 유저 그룹의 상당수는
+  Meetup 사이트를 홈페이지로 합니다. Meetup은 유저 그룹을 위한 몇 가지 툴을
```
[the remark this answers is not in the document]
Comment (riseshia) [reply ko/community/user-groups/index.md]: fixed e2ff452

File: ko/community/user-groups/index.md [hunk 4]
```diff
@@ -19,24 +19,25 @@ Ruby 유저 그룹은 전적으로 Ruby에만 전념합니다. 일반적으로 
...
+  제공하고 있습니다. 비공개 포럼, 공지를 위한 장소, 자동 리마인더, 훌륭한
+  RSVP(참석여부 체크) 시스템 등등이 그것이죠.
```
[the remark this answers is not in the document]
Comment (riseshia) [reply ko/community/user-groups/index.md]: fixed e2ff452

File: ko/community/user-groups/index.md (hunk 1 above)
Comment (riseshia) [reply ko/community/user-groups/index.md]: fixed e2ff452

**Review by yous — APPROVED** [review]
I'm OK with this.

File: ko/community/user-groups/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (yous) [reply ko/community/user-groups/index.md:26]: Ah, '전 세계' should be fixed.

[the remark this answers is not in the document]
Comment (riseshia) [reply ko/community/user-groups/index.md:26]:
```suggestion
- [rubyconferences.org/meetups][rc-meetups]는 전 세계의 Ruby 미트업 이벤트 일람을
```

[the remark this answers is not in the document]
Comment (riseshia) [reply ko/community/user-groups/index.md]: Fixed [1ea88a7](https://github.com/ruby/www.ruby-lang.org/pull/3296/commits/1ea88a7e8eab1186f1ea9a4df1bade91aa3c81a4)

[the remark this answers is not in the document]
Comment (yous) [reply ko/community/user-groups/index.md]: Thanks!

**Comment by riseshia** [comment]
Thanks for review~

**Pull request merged**